src/rpc/client.cpp

```diff
@@ -128,6 +128,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "importprivkey", 2, "rescan" },
     { "importaddress", 2, "rescan" },
     { "importaddress", 3, "p2sh" },
+    { "removeaddress", 2, "p2sh" },
```

Comment:
This is getting caught in the linter. Be sure the position of the argument is consistent with the dispatch table in rpcwallet.cpp (it isn't -- the index starts at 0).